### PR TITLE
Adds rag and others to website

### PR DESCRIPTION
**Pull request by CollectiveUnicorn**

### CODE REVIEW

**Review by justinthelaw — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ LeapfrogAI provides an API that closely matches that of OpenAI's. This feature a
 > | [whisper](https://github.com/defenseunicorns/leapfrogai-backend-whisper) | ✅ | 🚧 | ✅ | ✅ | ✅ | ✅ |
 > | [text-embeddings](https://github.com/defenseunicorns/leapfrogai-backend-text-embeddings) | ✅ | 🚧 | ✅ | ✅ | ✅ | ✅ |
 > | [vllm](https://github.com/defenseunicorns/leapfrogai-backend-vllm) | ✅ | ❌ | ✅ | ✅ | ✅ | ✅ |
+> | [rag](https://github.com/defenseunicorns/leapfrogai-backend-rag) | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ |
```
Comment (gphorvath) [inline README.md:61]: Doesn't RAG work on AMD64 and ARM64?

Comment (CollectiveUnicorn) [reply README.md:61]: I can double check, you might be right. I just wasn't sure.

Comment (gphorvath) [reply README.md:61]: I know we are pushing arm64 images, but you might want to run one to make sure 😁

Comment (CollectiveUnicorn) [reply README.md]: Yep it works

**Review by justinthelaw — APPROVED** [review]

**Review by gphorvath — APPROVED** [review]

**Pull request merged**